### ISSUE TITLE
"Sequence contains no elements" in EntityMapping.cs

### ISSUE THREAD

**Issue by mkrnic** [MoonStorm/Dapper.FastCRUD#75]
Line 91

     var maxColumnOrder = _propertyMappings.Select(propMapping => propMapping.ColumnOrder).Max();

This throws "Sequence contains no elements" if _propertyMappings has no values.

It won't have any values, for example, if your entire class is made of variables and not properties, as my coworker just found out :)

So maybe have a try..catch on that statement. On exception caught, rethrow with a more meaningful message, like "TEntity has no public properties defined".

**Comment by fgracin** [comment]
I coudn't agree with you more, @mkrnic.

**Comment by MoonStorm** [comment]
Yeah, I agree.  Maybe the check for public props be placed higher up.
